### PR TITLE
Image-to-pdf: Filename InputField is autofilled with the filename

### PR DESCRIPTION
**Pull request by sudeshim3**
FileName input field is auto-filled followed by _pdf suffix.
Say, if the file name is **android.jpg** the autofill text is **android_pdf**

# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java [hunk 1]
```diff
@@ -505,6 +510,32 @@ private void saveImagesInGrayscale() {
         }
     }
 
+    /**
+     * Returns the last file name obtained from filepath
+     */
+    private String getLastFileName(ArrayList<String> filesPath) {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java]: there are already a lot of functions in this file.
Can we move this and next function to FileUtils?

File: app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java [hunk 2]
```diff
@@ -505,6 +510,32 @@ private void saveImagesInGrayscale() {
...
+
+        if (filesPath.size() == 0)
+            return "";
+        String lastSelectedFilePath = filesPath.get(filesPath.size() - 1);
+        String nameWithoutExt = stripExtension(getFileNameWithoutExtension(lastSelectedFilePath));
+        return nameWithoutExt + getString(swati4star.createpdf.R.string.pdf_suffix);
+    }
+
+    private String stripExtension(String str) {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java]: please add javadoc style comments to newly added functions

File: app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java [hunk 3]
```diff
@@ -194,7 +195,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
     }
 
     @OnClick(R.id.tvNoOfImages)
-    public void onClick() { }
+    public void onClick() {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java]: why this change?

File: app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java (hunk 1 above)
Comment (sudeshim3) [reply app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java]: yes, moved to FileUtils.

File: app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java (hunk 2 above)
Comment (sudeshim3) [reply app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java]: yup added javadoc.

File: app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java (hunk 3 above)
Comment (sudeshim3) [reply app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java]: oops, sorry. before committing I had done "reformat code" which had added space between parentheses and moved the brace to the new line. I have reverted the code style the way it was before.

**Comment by Swati4star** [comment]
great @sudeshim3 👍

**Comment by Swati4star** [comment]
Merging this PR
but still https://github.com/Swati4star/Images-to-PDF/issues/779 is not yet resolved as it says for "text to PDF". @sudeshim3 you can work on this if you want.

**Comment by sudeshim3** [comment]
yup will work on it.
